src/lib/discord/bot.ts

```diff
@@ -13,13 +16,119 @@ new Command({
 		required: true,
 	} as const)
 	.handle(async i => {
+		const name: string = i.options.name
+		const entry: Entry | null = await fetchEntry(createSlug(name))
+		if (!entry) {
+			return {
+				content: `Requested entry "${name}" does not exist, or is unavailable`,
+				color: 16711680,
```

Comment:
This (and all the other instances of "16711680") could be changed to 0xff0000 to be more readable

Reply:
uhh, that's for the embed color that shows on the left.